test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/CustomSearchParamTests.cs

```diff
@@ -510,7 +511,7 @@ private async Task<(FhirResponse<Parameters> response, Uri uri)> RunReindexToCom
             return (response, reindexJobUri);
         }
 
-        private async Task<FhirResponse<Parameters>> WaitForReindexStatus(Uri reindexJobUri, params string[] desiredStatus)
+        internal static async Task<FhirResponse<Parameters>> WaitForReindexStatus(TestFhirClient client, Uri reindexJobUri, params string[] desiredStatus)
```

Comment:
Starting to look like an extension method :)

Reply:
I was at a loss for where to put it was my problem. I guess it would work in the TestFhirClient. Let me move it there.

Reply:
So move is done? You mean FhirClient.cs not TestFhirClient.cs?

Reply:
Yes, I had the wrong name.